### PR TITLE
optimized variables

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by alabulei1** [comment]
@DarumaDocker  Could you please check it out? We are using this code in production. So if you think this works well, please let me merge this PR.

**Comment by juntao** [comment]
I think we only need to respond to the `IssuesEvent` when a new issue is created. For example, if an issue is closed or re-opened, the bot should remain silent.

**Comment by alabulei1** [comment]
@juntao @DarumaDocker Please review again

File: src/lib.rs [hunk 1]
```diff
@@ -34,10 +41,16 @@ async fn handler(payload: EventPayload) {
         }
 
         EventPayload::IssuesEvent(e) => {
+            if e.action == IssuesEventAction::Closed {
```
Comment (juntao) [inline src/lib.rs:56]: Maybe this should be `!=Created` or something like that?

**Pull request merged**